aten/src/ATen/ScalarOps.h

```diff
@@ -4,23 +4,26 @@
 #include <ATen/Tensor.h>
 #include <ATen/Functions.h>
 
+namespace at {
+Tensor& scalar_fill(Tensor& self, Scalar value);
+TORCH_API Tensor scalar_tensor_static(Scalar s, const TensorOptions& options);
```

Comment:
Should these be in top level namespace too?

Reply:
If they are, they better have comments saying what they do! (But I suspect this was not intentional)

Reply:
Moved them to at::detail.